salt/modules/boto_iam.py

```diff
@@ -153,6 +153,297 @@ def role_exists(name, region=None, key=None, keyid=None, profile=None):
         return False
 
 
+def describe_role(name, region=None, key=None, keyid=None, profile=None):
+    '''
+    Get information for a role
+
+    CLI example::
+
+        salt myminion boto_iam.describe_role myirole
+    '''
+    conn = _get_conn(region, key, keyid, profile)
+    try:
+        info = conn.get_role(name)
+        if not info:
+            return False
+        return info
+    except boto.exception.BotoServerError as e:
+        log.debug(e)
+        msg = 'Failed to get {0} information.'
+        log.error(msg.format(name))
+        return False
```

Comment:
modules usually return either false , true of some data(true). thats what I got from writing them.